File: exercicios/R/1156/1156.r
input = file('stdin','r')

s = as.double(0)
j = as.double(1)
i = 1
while(i<40){
    s = as.double(s + as.double(i) / j)
    j = j * 2
    i = i + 2
}
write(sprintf("%.2f",s),'')

File: exercicios/JavaScript/1156/1156.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var s = 0, j=1;

for(let i=1; i<40;i+=2){
    s += parseFloat(i / j);
    j *= 2;
}
console.log(s.toFixed(2));